lib/patcher.js

```diff
@@ -340,7 +340,7 @@ function findChildReprints(newPath, oldPath, reprints) {
         if (k === "loc")
             continue;
 
-        if (!findAnyReprints(newPath.get(k), oldPath.get(k), reprints))
+        if (!findAnyReprints(newPath.get(k), oldPath.get(k), reprints, options))
```

Comment:
You're right! After I removed the overzealous reprinting it wasn't used anymore. Let me clean this up.